lib/core/utils/dq-element.js

```diff
@@ -1,8 +1,6 @@
 /*exported DqElement */
 
 function truncate(str, maxLength) {
-	'use strict';
```

Comment:
Babel puts us in strict mode everywhere. We don't need to do this anymore. I've been gradually removing them from functions I'm working on.